README.md

```diff
@@ -3,5 +3,60 @@
+|Client Management                   |Record the woman's pregnancy outcome                                                                                                                                                           |opensrp-client-core, opensrp-client-native-form                    |
+|Service Delivery Support                  |Monthly home visit schedules                                                                                                                                                      |opensrp-client-native-form (rules engine)                          |
+|Service Delivery Support                  |Schedules for high priority ANC services                                                                                                                                                      |opensrp-client-native-form (rules engine)                          |
+|Client Management            |Record services provided during monthly home visit                                                                                                                                  |opensrp-client-native-form (rules engine)                          |
+|Service Delivery Support            |Call the woman                                                                                                                                                                        |                                                                   |
+|Client Identification & Registration|Register the woman for PNC services after delivery outcome                                                                                                                                                                      |opensrp-client-core, opensrp-client-native-form                    |
+|Service Delivery Support                  |PNC visit schedules - at home and at health facility                                                                                                                                                      |opensrp-client-native-form (rules engine)                          |
+|Service Delivery Support                  |Schedules for high priority PNC services                                                                                                                                                      |opensrp-client-native-form (rules engine)                          |
+|Client Management            |Record services provided during PNC home visit                                                                                                                                  |opensrp-client-native-form (rules engine)                          |
+                                                                |opensrp-server-web, opensrp-client-core, opensrp-client-native-form|
```

Comment:
this line looks like it's missing the Function and Feature, do know what does are? Or is the whole line extraneous and these should be removed?

Reply:
Ah ok I think just remove it. Let me try to figure out how to do that from here.